packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -299,9 +299,8 @@ describe('FLE tests', () => {
     expect(compactResult).to.include('The "compactStructuredEncryptionData" command requires Mongo instance configured with auto encryption.');
   });
 
-  context('6.0+', () => {
-    skipIfServerVersion(testServer, '< 6.0'); // Queryable Encryption only available on 6.0+
-    skipIfServerVersion(testServer, '> 6.x'); // TODO(MONGOSH-1410): Queryable Encryption made a breaking change for 7.0
+  context('7.0', () => {
+    skipIfServerVersion(testServer, '< 7.0');
```

Comment:
Probably makes sense to keep this in the current format (with a basic comment explaining why it’s skipped):

```suggestion
  context('7.0+', () => {
    skipIfServerVersion(testServer, '< 7.0'); // Queryable Encryption v2 only available on 6.0+
```